run.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+BASE_DIR=$(pwd)
```

Comment:
This only works when running `run.sh` from the directory it's in. Use something like `cwd=$(cd $(dirname $0); pwd -P)` to get the real path.

Reply:
fixed.